lib/billy.rb

```diff
@@ -28,4 +28,18 @@ def self.proxy
   def self.certificate_authority
     @certificate_authority ||= Billy::Authority.new
   end
+
+  # This global shortcut can be used inside of request stubs. You can modify
+  # the request beforehand and/or modify the actual response which is passed
+  # back by this method. But you can also implement a custom proxy passing
+  # method if you like to. This is just a shortcut.
+  def self.pass_request(params, headers, body, url, method)
+      handler = proxy.request_handler.handlers[:proxy]
+      response = handler.handle_request(method, url, headers, body)
```

Comment:
nab: It's interesting to see the difference between internet/external APIs here.

Reply:
Yes because of:

```ruby
proxy.stub('http://example.com/').and_return(Proc.new { |*args|
  response = pass_request(*args)
end
```

Reply:
Makes it easier to use here without introduce a breaking change. Its more likely to intercept the response and modify it as to tweak the request. It was fine in first place, no need to debate on it in a fixup.